spec/views/spotlight/sir_trevor/blocks/_recent_items_block.html.erb_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'spotlight/sir_trevor/blocks/_recent_items_block.html.erb', type: :view do
+  let(:p) { 'spotlight/sir_trevor/blocks/recent_items_block.html.erb' }
+  let(:block) do
+    RecentItemsBlock.new({ type: 'block', data: {} }, view)
+  end
+  let(:exhibit) { FactoryBot.create(:exhibit) }
+
+  before do
+    # The view spec is separate from the CatalogController, which has this piece
+    # of configuration in Blacklight. So to get paths to resolve, we need this.
+    view.lookup_context.view_paths.push 'app/views/catalog'
+    view.class.include Blacklight::SearchHelper
+    allow(view).to receive_messages(recent_items_block: block)
+    allow(view).to receive_messages(
+      blacklight_config: CatalogController.blacklight_config,
+      current_exhibit: exhibit,
+      search_session: {},
+      current_search_session: {}
+    )
+
+    url = "https://hydra-dev.princeton.edu/concern/scanned_resources/1r66j1149/manifest"
+    FactoryBot.create(:iiif_resource, url: url, exhibit: exhibit, manifest_fixture: "1r66j1149-expanded.json", source_metadata_identifier: "12345678", spec: self)
+    FactoryBot.create(:iiif_resource, url: "https://hydra-dev.princeton.edu/concern/scanned_resources/44558d29f/manifest", exhibit: exhibit, manifest_fixture: "44558d29f.json", spec: self)
```

Comment:
It's sort of weird having one url defined above and other defined inside the parens. both lines are sort of long and it would be good to be able to see what's being defined; maybe just make them both multi-line?